src/Toast/ToastContent.tsx

```diff
@@ -37,39 +38,69 @@ const ToastContent = React.forwardRef<HTMLElement, IToastContentProps>(function
 
   const showCloseIcon = closeIcon !== false;
 
+  const toastActions = actions.map((action, index) => {
+    const { label, onClick = noop, ...buttonProps } = action;
+    return (
+      <Button
+        key={index}
+        m="5px"
+        color={theme.toastFg}
+        backgroundColor="rgba(138,155,168,0.25)"
+        border="transparent"
+        {...buttonProps}
```

Comment:
Seems like we let `css` from `buttonProps` slip through

Reply:
now merges if existing